cfgov/cfgov/urls.py

```diff
@@ -101,8 +101,10 @@
             permanent=True)),
     url(r'^know-before-you-owe/',
         include(SheerSite('know-before-you-owe').urls)),
-    url(r'^adult-financial-education/$', TemplateView.as_view(
-        template_name='/adult-financial-education/index.html')),
+    url(r'^adult-financial-education/(?P<path>.*)$',
+        RedirectView.as_view(
+            url='/practitioner-resources/adult-financial-education/%(path)s',
+            permanent=True)),
```

Comment:
Could we implement this as an Apache redirect instead of here in `urls.py`? This approach might lead to this file getting cluttered up over time.

Reply:
Alternatively we could implement this as a Wagtail redirect, I believe given the path.

I think no matter what we're going to have clutter, be it here, Apache, or in Wagtail. Apache would probably be the most performant place for it though.

Reply:
I had a laugh about that as well, but I've [moved it to the Apache config](https://github.cfpb.gov/cfpb-infra/ansible-consumer-finance/pull/144) for good measure.

Reply:
☝️ @jimmynotjim FYI I edited your comment to remove reference to GHE which shouldn't be in public repos.